magento/magento2
# Fix #29063 - Create configurations: not all configurable attributes are visible

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#29063

### Manual testing scenarios (*)

1. Create few product attributes of type Dropdown and scope Global
2. Please check fixed issue for next steps

### Questions or comments

I'm creating this PR without mftf test for now since I'm not able to create it at the moment - in case somebody from core team would have possibility to do it quicker than me.

### Contribution checklist (*)
